.github/workflows/test.yaml

```
@@ -5,16 +5,109 @@ on:
+          path: node_modules
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-
+
+      - name: Run ${{ matrix.tests.type }}
+        run: |
+          npm run test:ember -- ${{matrix.tests.commandArgs}} --path=dist --wmmf
+
+      - name: Job Summary
+        if: always()
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)